app/models/forms/consumer_candidate.rb

```diff
@@ -160,7 +160,8 @@ def age_less_than_18
       return unless (self.dob_check == "false" || self.dob_check.blank?) && self.dob.present?
 
       if ::TimeKeeper.date_of_record.year - self.dob.year < 18
-        errors.add(:base, "Please verify your date of birth. If it's correct, please continue.", :level => :warning)
+        message_key = EnrollRegistry.feature_enabled?(:bs4_consumer_flow) ? "insured.match_person.dob_valid_error_2" : "insured.match_person.dob_valid_error"
```

Comment:
bit of a nitpick, but i think it might be more readable if you had one translation string here and just passed into that a var for the name of the continue button.

Reply:
I've updated this error message to use the existing translations we have for continue and continue to next step. The only caveat is, the next step version is missing a "the", but I think it's worth it to omit that for simpler complexity here with respect to the mockups.